frontend/test/unit/specs/components/v-back-to-search-results-link.spec.js

```diff
@@ -0,0 +1,45 @@
+import { createLocalVue } from "@vue/test-utils"
+import { fireEvent, render } from "@testing-library/vue"
+
+import { PiniaVuePlugin, createPinia } from "~~/test/unit/test-utils/pinia"
+
+import { useAnalytics } from "~/composables/use-analytics"
+
+import VBackToSearchResultsLink from "~/components/VBackToSearchResultsLink.vue"
+
+jest.mock("~/composables/use-analytics", () => ({
+  useAnalytics: jest.fn(() => ({
+    sendCustomEvent: jest.fn(),
+  })),
+}))
+
+const localVue = createLocalVue()
+localVue.use(PiniaVuePlugin)
+describe("VBackToSearchResultsLink", () => {
+  it("should send analytics event when clicked", async () => {
+    const sendCustomEventMock = jest.fn()
+
+    useAnalytics.mockImplementation(() => ({
+      sendCustomEvent: sendCustomEventMock,
+    }))
```

Comment:
Doesn't this duplicate the mock in `jest.mock` call? It seems like one could be removed in favour of the other.

Reply:
Fixed :)